boards/arm/nucleo_l552ze_q/CMakeLists.txt

```diff
@@ -60,6 +66,6 @@ if (CONFIG_BUILD_WITH_TFM)
 		COMMAND ${CMAKE_COMMAND} -E copy ${CMAKE_BINARY_DIR}/tfm/bin/bl2.bin ${CMAKE_BINARY_DIR}/mcuboot.bin
 
 		#Execute post build script postbuild.sh
-		COMMAND ${CMAKE_BINARY_DIR}/tfm/postbuild.sh
+		COMMAND ${CMAKE_BINARY_DIR}/tfm/postbuild.sh ${COMPILER_FULL_PATH}
```

Comment:
This is a nice improvement.
Please also update the following note: https://github.com/zephyrproject-rtos/zephyr/blob/3ce7afdaaa005dc0f46d721ad77d0c275ce0df9f/samples/tfm_integration/tfm_ipc/README.rst#L132

Reply:
done :)